interface/forms/procedure_order/delete.php

```diff
@@ -0,0 +1,125 @@
+            "f.deleted = 0 AND " .
+            "lo.form_id = f.formdir AND lo.source = 'E' AND lo.uor > 0)",
+            array($pid, $encounter, $pid, $encounter)
+        );
+    }
+    // log the event
+    EventAuditLogger::instance()->newEvent("delete", $_SESSION['authUser'], $_SESSION['authProvider'], 1, "Form " . $_POST['formname'] . " deleted from Encounter " . $_POST['encounter']);
+
+    // redirect back to the encounter
+    $address = "{$GLOBALS['rootdir']}/patient_file/encounter/$returnurl";
+    echo "\n<script>top.restoreSession();window.location='$address';</script>\n";
+    exit;
+}
+?>
+<html>
+
+<head>
+    <?php Header::setupHeader(); ?>
+    <title><?php echo xlt('Delete Encounter Form'); ?></title>
+</head>
+
+<body>
+    <div class="container mt-3">
+        <div class="row">
+            <div class="col-12">
+                <h2><?php echo xlt('Delete Encounter Form'); ?></h2>
+                <form method="post" action="<?php echo $rootdir; ?>/forms/procedure_order/delete.php"
+                    name="my_form" id="my_form">
+                    <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+                    <?php
+                    // output each GET variable as a hidden form input
+                    foreach ($_GET as $key => $value) {
+                        echo '<input type="hidden" id="' . attr($key) . '" name="' . attr($key) . '" value="' . attr($value) . '"/>' . "\n";
+                    }
+                    ?>
+                    <input type="hidden" id="confirm" name="confirm" value="1" />
+
+                    <p>
+                    <?php
+                    $formdir = $_GET["formname"];
+                    $formName = getFormNameByFormdir($formdir);
+                    echo xlt('You are about to delete the following form from this encounter') . ': ' . text(xl_form_title($formName["form_name"]));
+                    ?>
+                    </p>
+                    <div class="btn-group">
+                        <button type="button" class="btn btn-danger btn-delete" id="confirmbtn" name="confirmbtn" value='<?php echo xla('Yes, Delete this form'); ?>'>
+                            <?php echo xlt('Yes, Delete this form'); ?>
+                        </button>
+                        <button type="button" class="btn btn-secondary btn-cancel" id="cancel" name="cancel" value='<?php echo xla('Cancel'); ?>'>
+                            <?php echo xlt('Cancel'); ?>
+                        </button>
+                    </div>
+                </form>
+            </div>
+        </div>
+    </div>
+<script>
+// jQuery stuff to make the page a little easier to use
+
+$(function () {
+    $("#confirmbtn").on("click", function() { return ConfirmDelete(); });
+    $("#cancel").on("click", function() { location.href='<?php echo "$rootdir/patient_file/encounter/$returnurl";?>'; });
```

Comment:
since javascript context would do following for escaping (note removal of the single quotes):
```
$("#cancel").on("click", function() { location.href=<?php echo js_escape("$rootdir/patient_file/encounter/$returnurl"); ?>; });
```